bundle/Resources/config/doctrine.yaml

```diff
@@ -0,0 +1,10 @@
+orm:
+    entity_managers:
+        default:
+            auto_mapping: false
+            mappings:
+                netgen_layouts.remote_media:
+                    type: annotation
+                    prefix: Netgen\Layouts\RemoteMedia\API\Values
+                    dir: "%kernel.root_dir%/../vendor/netgen/layouts-remote-media/lib/API/Values"
```

Comment:
kernel.project_dir should be used here.

`dir: "%kernel.project_dir%/vendor/netgen/layouts-remote-media/lib/API/Values"`

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-remote-media/pull/15/commits/bf5c334bf66ff0cb669d85f62174d003c17912df.